tsconfig.json

```diff
@@ -1,11 +1,17 @@
 {
   "compilerOptions": {
     "module": "commonjs",
-    "target": "es6",
+    "target": "es5",
     "noImplicitAny": true,
     "sourceMap": true,
     "declaration": true,
-    "outDir": "out"
+    "outDir": "out",
+    "lib": [
+      "dom",
```

Comment:
I'm wondering why `dom` needed here, since this library should be independent of any browser-specific logic?